configure.ac

```diff
@@ -36,7 +36,10 @@ dnl Checks for libraries.
 AC_CHECK_LIB(pam, pam_start)
 AC_CHECK_LIB(tac, tac_connect)
 AC_CHECK_LIB(crypto, MD5_Init)
-AC_CHECK_LIB(crypto, RAND_bytes)
+AC_CHECK_LIB(crypto, RAND_pseudo_bytes,
+            [AC_DEFINE([HAVE_RAND_PSEUDO_BYTES], [1], [RAND_pseudo_bytes is present])])
```

Comment:
Please change the last string to:

`[Define to 1 if you have the RAND_pseudo_bytes function.]`

for consistency with the rest of `config.h` as output.